invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/RecordsResultsListItem.js

```diff
@@ -135,6 +137,11 @@ class RecordsResultsListItem extends Component {
                   </small>
                 </p>
               )}
+              <div>
+                {`${uniqueViews} ${i18next.t("views")}, ${uniqueDownloads} ${i18next.t(
```

Comment:
Can we add singular/plural support? And also the variable should go inside the translated string, not via concatenation. We should always avoid concatenation, each language can have a different way to formulate a sentence, different order of words.

```suggestion
                {`${i18next.t("{{count}} views", {count: uniqueViews})}, ${i18next.t({{count}} downloads, {count: uniqueDownloads}}
```
Actually, ideally, it should be one string only:
`18next.t("{{uniqueViews}} views, ({{uniqueDownloads}} downloads, {uniqueViews: uniqueViews, uniqueDownloads: uniqueDownloads}}` but it looks like that [i18next requires {{count}}](https://www.i18next.com/translation-function/plurals), so it should be tested.

I see that in your screenshots you have skipped the `1` ;)

Reply:
I hoped you wouldn't catch me :unamused: 

Jokes aside, tbh I'm clueless about translations – thanks for the tip!
Will rework accordingly.

Reply:
Made it work with the `i18next` features for [plurals](https://www.i18next.com/translation-function/plurals) and [nesting](https://www.i18next.com/translation-function/nesting), and given that we're still on v20 I use the translation keys for the [v3 JSON](https://www.i18next.com/misc/json-format#i18next-json-v3).